src/D2L.Bmx/AwsCredsCreator.cs

```diff
@@ -30,12 +31,22 @@ bool cache
 			);
 		}
 
+		var accountSource = ParameterSource.CliArg;
 		if( string.IsNullOrEmpty( account ) && !string.IsNullOrEmpty( config.Account ) ) {
 			account = config.Account;
+			accountSource = ParameterSource.Config;
+		}
+		if( !string.IsNullOrEmpty( account ) && !nonInteractive ) {
+			consoleWriter.WriteParameter( ParameterDescriptions.Account, account, accountSource );
```

Comment:
seems reasonable to not print these out when user has explicitly specified `--non-interactive`?